src/main/webapp/logInPage.html

```diff
@@ -0,0 +1,25 @@
+<html>
+
+<head>
+  <link rel="stylesheet" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Ubuntu" rel="stylesheet">
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
+  <title>Sign in</title>
+</head>
+
+<body>
+  <div class="main">
+    <p class="sign" align="center">Sign in</p>
+    <form class="form1">
+      <input class="un " type="text" align="center" placeholder="Username">
```

Comment:
We usually want to keep our class-names relatively descriptive, so "username" is probably a better class than "un" here.